recipes/psi4/conda_build_config.yaml

```diff
@@ -0,0 +1,21 @@
+channel_sources:
+  - conda-forge/label/libint_dev,conda-forge
+channel_targets:
+  - conda-forge psi4_dev
+mkl:
+  - 2022.1.0  # [win]
+dftd4:
+  # provides b97-3c parameters
+  - 3.5.0
+libint:
+  # provides rt-switchable ordering (src) and derivative integrals (build)
+  - 2.7.3dev1
+pylibefp:
+  # 0.6.3 can detect py311
+  - 0.6.2
+geometric:
+  # new API
+  - "1.0"
+pymdi:
+  # breaking API change
+  - "1.2"
```

Comment:
Please only specify dependency versions in the conda_build_config if you want to build against multiple versions. Otherwise, if this package has requires specific versions for dependencies, name them in the meta.yaml.

In fact, we prefer that you don't specify your own constraints if the dependencies are already listed in the [channel-wide pinnings](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml).

Reply:
Ok, I moved all the version deps besides `mkl` to meta.yaml. I agree that the windows `mkl` pin is a little troubling -- it was the least disruptive route ("windows blas story").